modules/nginx/src/test/java/org/testcontainers/junit/SimpleNginxTest.java

```diff
@@ -8,52 +8,58 @@
 import org.testcontainers.containers.wait.strategy.HttpWaitStrategy;
 
 import java.io.*;
+import java.net.URL;
 import java.net.URLConnection;
 
-import static org.rnorth.visibleassertions.VisibleAssertions.assertTrue;
-import static org.rnorth.visibleassertions.VisibleAssertions.info;
-
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.rnorth.visibleassertions.VisibleAssertions.*;
 
 /**
  * @author richardnorth
  */
 public class SimpleNginxTest {
 
-    private static File contentFolder = new File(System.getProperty("user.home") + "/.tmp-test-container");
+    private static String tmpDirectory = System.getProperty("user.home") + "/.tmp-test-container";
 
+    // creatingContainer {
     @Rule
     public NginxContainer nginx = new NginxContainer<>()
-        .withCustomContent(contentFolder.toString())
+        .withCustomContent(tmpDirectory)
         .waitingFor(new HttpWaitStrategy());
+    // }
 
     @SuppressWarnings({"Duplicates", "ResultOfMethodCallIgnored"})
     @BeforeClass
-    public static void setupContent() throws FileNotFoundException {
+    public static void setupContent() throws Exception {
+        // addCustomContent {
+        // Create a temporary dir
+        File contentFolder = new File(tmpDirectory);
         contentFolder.mkdir();
-        contentFolder.setReadable(true, false);
```

Comment:
I couldn't find a reason  why these were being set. I'm happy to put them back but for the purposes of a test/demo I don't think they are required.